lib/common/client/concerns/service_errors.rb

```diff
@@ -33,7 +33,7 @@ def handle_error(error)
     end
 
     def log_original_error(error)
-      level, message = if (400...500).cover? error.status
+      level, message = if (401...500).cover? error.status
```

Comment:
Why did you change this line?

Reply:
ack, that should be 402...500, but the intent was to move 401 out of the expected response codes to use the unexpected method.

Reply:
I personally find it maddening that ruby has a `..` and `...` operator.